From ee28ae84977fd1c0b13c04cde2c3fb766aab819c Mon Sep 17 00:00:00 2001
From: Jun Li <r65092@freescale.com>
Date: Thu, 11 Jun 2009 17:01:43 +0800
Subject: [PATCH] ENGR00113254 build in usb drivers in default for i.MX51

Change default menuconfig of fsl_otg_arc, ehci-hcd,
arcotg_udc drivers to be built in.

Signed-off-by: Li Jun <r65092@freescale.com>
---
 arch/arm/configs/imx51_defconfig |   11 ++++++-----
 1 files changed, 6 insertions(+), 5 deletions(-)

diff --git a/arch/arm/configs/imx51_defconfig b/arch/arm/configs/imx51_defconfig
index bc7bbf1..0f1c8df 100644
--- a/arch/arm/configs/imx51_defconfig
+++ b/arch/arm/configs/imx51_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.28
-# Fri Jun 12 16:38:25 2009
+# Thu Jun 11 16:39:49 2009
 #
 CONFIG_ARM=y
 CONFIG_SYS_SUPPORTS_APM_EMULATION=y
@@ -206,6 +206,7 @@ CONFIG_ARCH_MXC_HAS_NFC_V3_2=y
 CONFIG_MXC_TZIC=y
 CONFIG_DMA_ZONE_SIZE=64
 CONFIG_UTMI_MXC=y
+CONFIG_UTMI_MXC_OTG=y
 
 #
 # Processor Type
@@ -1259,8 +1260,8 @@ CONFIG_USB=y
 # CONFIG_USB_DEVICEFS is not set
 CONFIG_USB_DEVICE_CLASS=y
 # CONFIG_USB_DYNAMIC_MINORS is not set
-# CONFIG_USB_SUSPEND is not set
-# CONFIG_USB_OTG is not set
+CONFIG_USB_SUSPEND=y
+CONFIG_USB_OTG=y
 # CONFIG_USB_OTG_WHITELIST is not set
 # CONFIG_USB_OTG_BLACKLIST_HUB is not set
 # CONFIG_USB_MON is not set
@@ -1357,7 +1358,7 @@ CONFIG_USB_STORAGE=y
 # CONFIG_USB_IOWARRIOR is not set
 # CONFIG_USB_ISIGHTFW is not set
 # CONFIG_USB_VST is not set
-CONFIG_USB_GADGET=m
+CONFIG_USB_GADGET=y
 # CONFIG_USB_GADGET_DEBUG_FILES is not set
 CONFIG_USB_GADGET_VBUS_DRAW=2
 CONFIG_USB_GADGET_SELECTED=y
@@ -1375,7 +1376,7 @@ CONFIG_USB_GADGET_SELECTED=y
 # CONFIG_USB_GADGET_NET2280 is not set
 # CONFIG_USB_GADGET_GOKU is not set
 CONFIG_USB_GADGET_ARC=y
-CONFIG_USB_ARC=m
+CONFIG_USB_ARC=y
 # CONFIG_USB_GADGET_DUMMY_HCD is not set
 CONFIG_USB_GADGET_DUALSPEED=y
 CONFIG_USB_GADGET_ARC_OTG=y
-- 
1.5.4.4

